pkg/volume/flexvolume/flexvolume.go

```diff
@@ -98,6 +98,10 @@ func (plugin *flexVolumePlugin) RequiresRemount() bool {
 	return false
 }
 
+func (plugin *flexVolumePlugin) SupportsMountOption() bool {
+	return false
```

Comment:
do not block merge on this comment, but isn't flex on where we should support mount options? Flex is the place we have absolutely no idea at all what people need, right?